fastserde/avro-fastserde/build.gradle

```diff
@@ -6,6 +6,40 @@
 
 plugins {
   id "java-library"
+  id "java-test-fixtures"
+}
+
+//Disable fixture publishing
+components.java.withVariantsFromConfiguration(configurations.testFixturesApiElements) { skip() }
+components.java.withVariantsFromConfiguration(configurations.testFixturesRuntimeElements) { skip() }
+
+//This block is only supported and required when building with JDK11
+if (JavaVersion.current() == JavaVersion.VERSION_11) {
+  sourceSets {
```

Comment:
this section (test fixtures in this module) only exists for the unit test modules to pull java11-specific code, and not for any unit testing in this module. could you add documentation explaining what its doing?

Reply:
Text fixtures are now removed, and it is just a custom source set